restcomm/restcomm.dao/src/main/java/org/restcomm/connect/dao/entities/SmsMessage.java

```diff
@@ -47,13 +46,20 @@
     private final Direction direction;
     private final BigDecimal price;
     private final Currency priceUnit;
+    private final String tlvTag;
```

Comment:
@scottbarstow I do agree, currently as per your comment on Slack the SmsService only fetches tlv 1550 which is why its only adding one tlv. However the code youre pointing at is the for-test-only Sms REST API tlv modification which im going to clean out.